lib/volume-parser.js

```diff
@@ -0,0 +1,29 @@
+exports.parseVolume = (fieldTagV) => {
+  const regExp = [/(?:^|\s|-)(?:volume|vol\.?|v\.?)\s?(\d*)[-|/]?(\d*)?/gi, /(?:^|\s|-)(?:no\.?)\s?(\d*)[-|/]?(\d*)?/gi, /(?:^|\s|-)(?:bd\.?)\s?(\d*)[-|/]?(\d*)?/gi, /(?:^|\s|-)(?:reel|r\.?)\s?(\d*)[-|/]?(\d*)?/gi, /(?:^|\s|-)(?:reel|r\.?)\s?(\d*)[-|/]?(\d*)?/gi]
```

Comment:
Can you format this with line-breaks and inline comments about generally what each is supposed to match and or an example?